large-hashable.cabal

```diff
@@ -22,35 +22,43 @@ library
                      , Data.LargeHashable.Class
                      , Data.LargeHashable.MD5
                      , Data.LargeHashable.Intern
-                     , Data.LargeHashable.Serial
+                     , Data.LargeHashable.LargeWord
                      , Data.LargeHashable.TH
-  build-depends:       base
+  build-depends:       aeson
+                     , base
                      , text
                      , bytestring
                      , transformers
                      , base16-bytestring
                      , bytes
                      , containers
                      , unordered-containers
+                     , scientific
+                     , strict
                      , time
                      , template-haskell
+                     , utf8-light
+                     , vector
+                     , void
   default-language:    Haskell2010
   ghc-options:         -optc -O3 -fno-cse -W -fwarn-unused-imports -fwarn-unused-binds
                        -fwarn-unused-matches -fwarn-unused-do-bind -fwarn-wrong-do-bind
-                       -pgmPcpphs -optP--cpp -rtsopts -threaded -funbox-strict-fields
-                       -fwarn-missing-signatures
+                       -pgmPcpphs -optP--cpp -rtsopts -threaded -fno-warn-name-shadowing
+                       -fwarn-missing-signatures -Werror -O2
```

Comment:
`-Werror` is not a good idea for a library to be distributed via hackage. You can still enforce it while developing using `stack build --pedantic`